lib/repeatable.js

```diff
@@ -74,6 +74,9 @@ module.exports = function(Queue) {
               name,
               data,
               _.extend(_.clone(opts), {
+                repeat: _.extend(_.clone(opts.repeat), {
```

Comment:
This starting to get quite "hairy" with so many extends and clones. I think we can do better, maybe something like (untested):
```
return Job.create(
              _this,
              name,
              data,
              _.defaultsDeep( {
                repeat: {
                  count: currentCount
                },
                jobId: customId,
                delay: delay < 0 ? 0 : delay,
                timestamp: now,
                prevMillis: nextMillis
              }, opts)
            );
```

Reply:
I didn't know there was a defaultsDeep in lodash. Good to know :)

I changed it and also added a test case to support my scenario and a documentation for the count attribute on the RepeatOpts interface.

Reply:
I didn't know there was a defaultsDeep in lodash. Good to know :)

I changed it and also added a test case to support my scenario and a documentation for the count attribute on the RepeatOpts interface.

Reply:
I didn't know there was a defaultsDeep in lodash. Good to know :)

I changed it and also added a test case to support my scenario and a documentation for the count attribute on the RepeatOpts interface.